ios/browser/providers/BUILD.gn

```diff
@@ -17,6 +17,7 @@ group("brave_providers") {
     "//ios/chrome/browser/providers/branded_images:chromium_branded_images",
     "//ios/chrome/browser/providers/discover_feed:chromium_discover_feed",
     "//ios/chrome/browser/providers/lens:chromium_lens",
+    "//ios/chrome/browser/providers/mailto_handler:chromium_mailto_handler",
```

Comment:
cc @kylehickinson - want to make sure this doesn't affect any handlers in the main brave ios app

Reply:
We should be fine, this provider only handles the actual mailto schemes when handled but it needs to be setup and called from the iOS side which we dont do